Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'support'
+DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
+
+node('mongodb-2.4') {
```

Comment:
https://ci.dev.publishing.service.gov.uk/job/govuk_support/configure doesn't restrict where this project can be built so we don't need to restrict it to a mongodb-2.4 node here. Should be:
```
node {
...
}
```